src/platform/native.zig

```diff
@@ -204,7 +204,9 @@ pub const Platform = struct {
         if (builtin.os.tag == .linux and platform.linux_gamemode_is_active) {
             const gamemode = @import("gamemode");
             gamemode.requestEnd() catch |err| {
-                std.log.err("Gamemode error {} -> {s}", .{ err, gamemode.errorString() });
+                if (!std.mem.containsAtLeast(u8, gamemode.errorString(), 1, "dlopen failed")) {
```

Comment:
Maybe add comment here:

```
// We don't want these warnings to appear if you don't have gamemode installed, so do a simple string check:
//
// error: Gamemode error error.RequestFailed -> dlopen failed - libgamemode.so: cannot open shared object file: No such file or directory
```

But also, is this the right place for this actually ? I'm surprised it is in the `deinitLinuxGamemode` function and not `initLinuxGamemode` that causes this error to get printed